js/contributor-cards/generate-contributor-lists.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+# requires `bash, `jq` and `curl`
```

Comment:
Please add a copyright header to this file as I did with this file:
https://github.com/jakartaee/jakarta.ee/blob/src/build.sh#L2

Please make sure to replace my name with yours in the contributor's list!

Reply:
done